src/services/api/index.ts

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+export * from './request';
+
+const {
+  API_URL = 'https://utility-manager.herokuapp.com/api',
```

Comment:
I think we need to put this in `.env` variable such that one can easily specify which URL he wants to use based on the environment(ie `development`, `staging` and `production` would all have different URLs)